.circleci/config.yml

```diff
@@ -233,17 +233,14 @@ jobs:
 
   unit_test:
     docker:
-      - image: python:3.10.5-bullseye@sha256:dac61c6d3e7ac6deb2926dd96d38090dcba0cb1cf9196ccc5740f25ebe449f50
+      - image: cimg/python:3.11.2
```

Comment:
There is, but I thought it'd be good to make sure we use the same patch version that we use in our Dockerfile. Or is that being too careful?

Reply:
Being careful is good :) I was thinking that we could have a test that verifies that both versions are in sync. I doubt that dependabot will upgrade this line. Will it?

Reply:
Yeah, I don't think it will update this line. 

We're actually in an interesting situation now where there's a `python:3.11.3` published, but there isn't a `cimg/python:3.11.3` yet

Reply:
https://github.com/mozilla/remote-settings/issues/406